.gitignore

```diff
@@ -10,6 +10,7 @@ webapp/yarn-error.log
 mattermost.mattermost-license
 config/mattermost.mattermost-license
 config/config.json
+config/*.crt
```

Comment:
What is this we are adding to gitignore?

Reply:
It's for the Keycloak SAML setup. It's not related to this PR but I think it would be useful.